browser/browser.go

```diff
@@ -560,20 +570,6 @@ func (bow *Browser) Find(expr string) *goquery.Selection {
 	return bow.state.Dom.Find(expr)
 }
 
-// -- Unexported methods --
```

Comment:
I see this "Unexported methods" comment in one other place in the code.  It doesn't seem to be a widespread pattern.  I don't see it as practical to separate exported/unexported code like this,  but I have no problem putting it back. @lxt2